backend/src/Service/Subscription/SubscriptionService.php

```diff
@@ -83,7 +86,7 @@ public function packageBalance($storeOwner):RemainingOrdersResponse|string
      * @param storeOwner
      * @return string
      */
-    public function checkSubscription($storeOwner):string
+    public function checkSubscription($storeOwner): string
     {
        $checkSubscription = $this->checkValidityOfSubscription($storeOwner);
```

Comment:
What is the use of this statement, if we don't use its returned result?

Reply:
Will be used later, but now running in the background is very important when called.